spec/active_record_with_lock_spec_helper.rb

```diff
@@ -28,3 +28,11 @@ def active_record_with_lock_adapter_connection
     lock_connection_pool: LockDatabaseRecord.connection_pool,
   )
 end
+
+RSpec.configure do |config|
+  if ENV["ADAPTER"] == "ActiveRecordWithLock"
```

Comment:
What's the reason for gating this test? Also, if we're going to gate it do we need to document it/specify it in CI ?